tee-worker/litentry/core/assertion-build/src/a6.rs

```diff
@@ -40,39 +34,29 @@ const VC_A6_SUBJECT_TAG: [&str; 1] = ["Twitter"];
 ///    * 1,000+ followers
 ///    * 10,000+ followers
 ///    * 100,000+ followers
-pub fn build(
-	identities: Vec<Identity>,
-	shard: &ShardIdentifier,
-	who: &AccountId,
-) -> Result<Credential> {
-	debug!(
-		"Assertion A6 build, who: {:?}, identities: {:?}",
-		account_id_to_string(&who),
-		identities
-	);
+pub fn build(req: &AssertionBuildRequest) -> Result<Credential> {
+	debug!("Assertion A6 build, who: {:?}", account_id_to_string(&req.who),);
```

Comment:
We don't want to have such sensitive information to appear in logs or is it jus irrevelant ?